cfgov/ask_cfpb/fixtures/glossary_terms.csv

```diff
@@ -0,0 +1 @@
Payday loans,Payday loan,1567,Préstamo del día de pago
Prepaid cards,Additional card fee,,
Prepaid cards,ATM balance inquiry fee,,
Prepaid cards,Bill payment fee,,
Prepaid cards,Card cancellation fee,,
Prepaid cards,Card replacement fee,,
Prepaid cards,Card-to-card transfer fee,,
Prepaid cards,Cash reload fee,,
Prepaid cards,Closed-loop prepaid card,,
Prepaid cards,Decline fee,,
Prepaid cards,Flexible spending account (FSA) or health savings account (HAS) card,417,Tarjeta de gasto flexible (FSA) o de ahorros para gastos de salud (HAS)
Prepaid cards,Foreign transaction fee,,
Prepaid cards,Government benefit card,409,Tarjeta de beneficios gubernamentales
Prepaid cards,In-network/out of network fees,,
Prepaid cards,Inactivity fee,,
Prepaid cards,Monthly fee,,
Prepaid cards,Open-loop prepaid card,,
Prepaid cards,Paper statement fee,,
Prepaid cards,Payroll card,401,Tarjeta de nómina
Prepaid cards,Per purchase fee,,
Prepaid cards,Prepaid Card,379,Tarjeta prepagada
Prepaid cards,Reload pack,451,Paquete de recarga
Prepaid cards,Reloadable prepaid card,,
Prepaid cards,Text alert service,507,Servicios de alertas de texto
Prepaid cards,Transaction fee,,
Reverse mortgages,Good Faith Estimate,146,Estimado de Buena Fe
Reverse mortgages,Reverse mortgage,224,Hipoteca inversa
Student loans,Co-signer,565,Cofirmante
Student loans,Co-signer release,,
Student loans,Cost of attendance,,
Student loans,Default,,
Student loans,Deferment,629,Diferimiento
Student loans,Direct PLUS loan,553,"Préstamo ""Direct PLUS"""
Student loans,Discharge due to disability or death,,
Student loans,Extended repayment,,
Student loans,FAFSA¨,,
Student loans,Federal Direct loan,1553,Préstamo directo federal
Student loans,Federal student loans,,
Student loans,Forbearance,631,Aplazamiento
Student loans,GI Bill®,,
Student loans,Graduated repayment,635,Plan de pago graduado
Student loans,Grant,,
Student loans,Income-based repayment ,633,Plan de pago basado en ingresos
Student loans,Income-driven repayment,,
Student loans,Loan consolidation,,
Student loans,National Student Loan Data System (NSLDS),589,Sistema Nacional de Información de Préstamos para Estudiantes (NSLDS)
Student loans,Pay As You Earn (PAYE),1555,"Plan Pague Mientras Gane (Pay as you Earn, PAYE)"
Student loans,Perkins loan,551,Préstamo Perkins
Student loans,Private or alternative student loans,559,Préstamos estudiantiles privados o alternativos
Student loans,Private student loans,,
Student loans,Public Service Loan Forgiveness,641,Perdón de préstamo por servicio público
Student loans,Servicemembers Civil Relief Act (SCRA),,
Student loans,Stafford loan,549,Préstamos Stafford
Student loans,Standard repayment,,
Student loans,Student loan servicer,583,Administrador de préstamos estudiantiles
Student loans,Subsidized loan,573,Préstamo subsidiado
Student loans,Total loan balance,,
Student loans,Tuition repayment plan,563,Plan de pago de matrícula
Student loans,Unsubsidized loan,,
Student loans,Work study,,
```

Comment:
I wonder if it would be friendlier to include the header in the CSV rather than adding it later.
Also breaking each line would make the CSV reviewable, especially if there are any changes.

Reply:
Thanks for the help on this, @higs4281! Fixed in the latest.